### PR TITLE
Aggregate version

### PR DESCRIPTION
**Pull request by pglushenkov**

### CODE REVIEW

File: packages/storage-drivers/resolve-storage-mongo/src/index.js [hunk 1]
```diff
@@ -33,7 +42,13 @@ function createDriver({ url, collection }) {
     }
 
     return {
-        saveEvent: event => getCollection().then(coll => coll.insert(event)),
+        saveEvent: event =>
+            getCollection().then(coll => coll.insert(event)).catch((e) => {
+                if (e.code === DUPLICATE_KEY_ERROR) {
```
Comment (IhostVlad) [inline packages/storage-drivers/resolve-storage-mongo/src/index.js]: Should this change be reflected in related tests: https://github.com/reimagined/resolve/blob/ae633ee345a18cbcb2c251716ad22d8aede2cf1b/packages/storage-drivers/resolve-storage-mongo/test/index.tests.js ?

File: packages/storage-drivers/resolve-storage-mongo/src/index.js [hunk 2]
```diff
@@ -25,6 +28,12 @@ function createDriver({ url, collection }) {
                     coll
                         .createIndex('timestamp')
                         .then(() => coll.createIndex('aggregateId'))
+                        .then(() =>
+                            coll.createIndex(
```
Comment (IhostVlad) [inline packages/storage-drivers/resolve-storage-mongo/src/index.js]: Should this change be reflected in related tests: https://github.com/reimagined/resolve/blob/aggregate_version/packages/storage-drivers/resolve-storage-mongo/test/index.tests.js ?

File: packages/storage-drivers/resolve-storage-mongo/src/index.js (hunk 1 above)
Comment (pglushenkov) [reply packages/storage-drivers/resolve-storage-mongo/src/index.js]: ok

File: packages/storage-drivers/resolve-storage-mongo/src/index.js (hunk 2 above)
Comment (pglushenkov) [reply packages/storage-drivers/resolve-storage-mongo/src/index.js]: ok

**Pull request merged**